eg/Classes/Text.py

```diff
@@ -24,6 +24,8 @@
 
 class Default:
     class General:
+        deleteConfigMessage = "Do you want to remove all configuration data?"
```

Comment:
If i first got that dialog i was scared "what? delete all my config?! why?!". Then i realized it was just for the plugin. So it would be better to mention that:
`deleteConfigMessage = "Do you want to remove all configuration data for plugin '%s'?"`